controllers/managedclusterForCGU_controller.go

```diff
@@ -134,13 +131,19 @@ func (r *ManagedClusterForCguReconciler) Reconcile(ctx context.Context, req ctrl
 		}
 	} else {
 		// availableCondition is false or unknown, cluster is not ready
-		r.Log.Info("cluster is not ready", "RequeueAfter:", clusterStatusCheckRetryDelay)
```

Comment:
do you want to add this log to the new common function, probably helps with debugging?

Reply:
Potentially. Normally this log is just flooding our log file, especially with this change. There seem to be only 2 levels with our logger: info and error.